src/site/content/en/lighthouse-accessibility/focus-traps/index.md

```diff
@@ -1,9 +1,11 @@
 ---
 layout: post
```

Comment:
This post starts by mentioning the web aim checklist. But that might seem a bit odd because I don't think we mention it elsewhere.

Reply:
Agree. Don't see any reason why we need to cite our source in this one particular case and no others.